### PR TITLE
ocdm: Avoid crash with uninitialized plugin

### PR DESCRIPTION
**Pull request by calvaris**

### CODE REVIEW

**Comment by calvaris** [comment]
Can we please get this merged?

**Review by pwielders — APPROVED** [review]

**Comment by pwielders** [comment]
@calvaris we can only merge this if you sign the CLA...
follow the ink in the CLA: Copied her for your convenience...

File: Source/ocdm/open_cdm_impl.h [hunk 1]
```diff
@@ -80,7 +80,9 @@ struct OpenCDMAccessor : public OCDM::IAccessorOCDM {
             ASSERT(_remote != nullptr);
 
             if (_remote == nullptr) {
-                _client.Release();
+                if (_client.IsValid()) {
```
Comment (HaseenaSainul) [inline Source/ocdm/open_cdm_impl.h:83]: Think it is better if do this check before accessing the _client instance, i.e, before calling _client->IsOpen()

**Comment by calvaris** [comment]
I already signed the CLA.

**Pull request merged**